swagger.json

```diff
@@ -709,7 +709,42 @@
           }
         }
       }
```

Comment:
Can you change the host to `champs-bn-api.herokuapp.com`

Reply:
Feedback implemented!